yt_dlp/extractor/bilibili.py

```diff
@@ -535,6 +539,83 @@ def _real_extract(self, url):
         return self.playlist_result(self._entries(list_id), list_id)
 
 
+# https://www.bilibili.com/v/kichiku/mad/#/
+class BilibiliCategoryIE(SearchInfoExtractor):
+    IE_NAME = "Bilibili category extractor"
+    _MAX_RESULTS = 1000000
+    _VALID_URL = r'https?:\/\/www\.bilibili\.com\/v\/[a-zA-Z]+\/[a-zA-Z]+'
```

Comment:
Python's `r` string and `re` module does not require escaping `/`es. It's JavaScript thing.